.travis.yml

```diff
@@ -1,3 +1,8 @@
+dist: trusty
```

Comment:
I really don't like how everything in Travis is implicit. Specify `sudo: false` to run in containers, specify `dist: trusty` to run on GCE, etc. Until it no longer applies and you have a random setting in your Travis config that either doesn't do anything or does something different than what you initially intended.